examples/navier-stokes/navierstokes.c

```diff
@@ -647,7 +647,7 @@ int main(int argc, char **argv) {
   if (!icsfp)
       return CeedError(ceed, 1, "Function not found in the list");
   char str[256] = __FILE__":ICs";
```

Comment:
And there is `PETSC_MAX_PATH_LEN`, which may be better than a magic number.

Reply:
Updated